float32 rads_per_second
---
bool success